outplacement_deviation_report/wizards/deviation_report_wizard.py

```diff
@@ -110,5 +157,10 @@ def action_send(self):
             payload=json.dumps(payload),
             headers=api.get_headers(),
             params=querystring)
-        if response.status_code != 200:
+        if response.status_code not in (200, 201):
+            _logger.warning(response.text)
+            _logger.warning(response.status_code)
```

Comment:
Do we need to log the responses?

Reply:
Yep, if something has gone wrong we want to see why, its only when something has gone wrong we log this